### ISSUE TITLE
Export (save) dialog doesn't respect iOS safe area

### ISSUE THREAD

**Issue by tomayac** [excalidraw/excalidraw#4833]
Note how the `Export` string is hidden under the notch. Worse: the back button is hidden under the `Carrier`, which becomes the `Back to previous app` button, so if you came to Excalidraw from another app (or launched it via search), you can effectively never go back and leave the dialog. 

![excalidraw com_(iPhone X)](https://user-images.githubusercontent.com/145676/155087006-bff301b6-9c12-49f2-bb77-2846dc9364e2.png)

**Comment by FlynnFc** [comment]
I've moved the Dialog title to the bottom to support any mobile users with any sort of notch. 

Screenshot :
![example screenshot](https://user-images.githubusercontent.com/88561679/176538874-2589fc25-a5c3-443a-aefa-a49980af4847.png)

Example  vid :
https://user-images.githubusercontent.com/88561679/176537943-c1febcaa-d661-4e97-af14-92fa8e36fc48.mp4

Code : 
![excalidrawfix](https://user-images.githubusercontent.com/88561679/176538660-b3b16bfd-956e-43ec-acf5-aa7e8708f4ae.png)

**Comment by ad1992** [comment]
Yeah I like it too, might need to do some minor design tweaks but otherwise it looks good

**Comment by FlynnFc** [comment]
What changes did you have in mind? @ad1992

**Comment by JuanPedroPontVerges** [comment]
Isnt this closed? https://github.com/excalidraw/excalidraw/pull/5384

**Comment by tech-dipesh** [comment]
I think the issue is fixed, so it is better to close the issue whose issue is solved.
